### PR TITLE
Updated consumable potency / resilience values.

### PR DESCRIPTION
**Pull request by Phoenica**
related to #120

### CODE REVIEW

File: scripts/Addicted/System.reds [hunk 1]
```diff
@@ -192,24 +192,25 @@ public class AddictedSystem extends ScriptableSystem {
       E(s"no consumption tracked during prologue");
       return;
     }
-    let before: Threshold;
-    let after: Threshold;
-    let amount: Int32;
+    let previousThreshold: Threshold;
+    let newConsumption: Int32;
     let hint: Bool;
-    if Generic.IsAddictive(id) {
```
Comment (Roms1383) [inline scripts/Addicted/System.reds:199]: could you please avoiding renaming the vars when it's not necessary, it avoids "noise" in the review :)

**Review by Roms1383 — CHANGES_REQUESTED** [review]

File: scripts/Addicted/System.reds (hunk 1 above)
Comment (Phoenica) [reply scripts/Addicted/System.reds]: reverted :D

File: scripts/Addicted/Helper.reds [hunk 2]
```diff
@@ -14,32 +14,34 @@ public func IsLanguageSupported(locale: CName) -> Bool {
...
+    let resilienceModifier = Min(5, Max(1, daysSinceLastConsumed)); 
+
+    switch(consumableName) {
+      case Consumable.BlackLace: 
+      case Consumable.Alcohol: 
+        return resilienceModifier * 1;
     }
-    return 2;
+    return resilienceModifier * 2;
   }
```
Comment (Roms1383) [inline scripts/Addicted/Helper.reds:45]: I feel like your `Resilience` update would make things a bit too easy, what about applying a lower modifier in `Hard` mode (see `Config.reds`)?

**Review by Roms1383 — COMMENTED** [review]
Another little nit I'd like to review together before validating the PR

File: scripts/Addicted/Helper.reds (hunk 2 above)
Comment (Phoenica) [reply scripts/Addicted/Helper.reds:45]: Hm, so maybe dividing resilienceModifier in half?

**Review by Roms1383 — APPROVED** [review]
LGTM

**Pull request merged**